src/beam.cpp

```diff
@@ -207,7 +207,7 @@ void BeamSegment::CalcSetStemValues(Staff *staff, Doc *doc, BeamDrawingInterface
             }
             // handle cross-staff fTrem cases
             const auto [beams, beamsFloat] = beamInterface->GetFloatingBeamCount();
-            if ((coord->GetStemDir() == STEMDIRECTION_down) && ((beams > 0) || (beamsFloat > 0))) {
+            if ((coord->m_stem->GetDrawingStemDir() == STEMDIRECTION_down) && ((beams > 0) || (beamsFloat > 0))) {
```

Comment:
It is not impossible to have rests within a mixed beam. You need to check that `m_stem` is set before accessing it.

Reply:
Sorry, I am noticing we are filtering by chords and notes above. But still, it would be best to check it